packages/blade/src/components/BottomSheet/_decisions/decisions.md

```diff
@@ -0,0 +1,296 @@
+| onDismiss       | `Callback`                 | `undefined`         | called when the bottom sheet is closed, either by user state, hitting `esc` or tapping backdrop                                                                            |          |
+| initialFocusRef | `React.Ref`                | `undefined`         | ref element you want to get keyboard focus when opening the sheet                                                                                                          |          |
+| children | `React.ReactNode`                | `undefined`         | Accepts other BottomSheet sub components like BottomSheetHeader,BottomSheetBody,BottomSheetFooter                                                                                                          |          |
+
+### `BottomSheetBody` API
+
+| Prop       | Type              | Default     | Description                 | Required |
+| ---------- | ----------------- | ----------- | --------------------------- | -------- |
+| `children` | `React.ReactNode` | `undefined` | Contents of the BottomSheet | ✅       |
+
+### `BottomSheetHeader` API
+
+| Prop      | Type            | Default     | Description                           | Required |
+| --------- | --------------- | ----------- | ------------------------------------- | -------- |
+| `title`   | `string`        | `undefined` | Title of the Header                   |       |
+| `subtitle`   | `string`        | `undefined` | Subtitle of the Header                   |       |
+| `leading` | `React.ReactNode` | `undefined` | leading asset or icon to be placed at the left most side of the BottomSheetHeader |          |
+| `trailing` | `Badge, Link, Text, IconButton`, `undefined` | undefined  |     trailing component to be placed at the right most side of the BottomSheetHeader     |    |
+| `titleSuffix` | `Counter` | `undefined` | A component to be placed adjacent to the title text |          |
+| `hideDivider` | `boolean` | `false` | Show or hide the divider |          |
+| `showBackButton` | `boolean` | `false` | Show or hide back button |          |
+| `onBackButtonClick` | `boolean` | `false` | Event handler for the back button |          |
+
+### `BottomSheetFooter` API
+
+
+| Prop       | Type              | Default     | Description                 | Required |
+| ---------- | ----------------- | ----------- | --------------------------- | -------- |
+| `children` | `React.ReactNode` | `undefined` | Contents of the BottomFooter | ✅       |
+
+## Composition with Dropdown
+
+We will export `BottomSheet` component separately as an independant component but generally the pattern will be to use it with SelectInput, where in mobile devices the Select's dropdown will be replaced by the BottomSheet.
```

Comment:
```suggestion
We will export `BottomSheet` component separately as an independent component but generally the pattern will be to use it with SelectInput, where in mobile devices the Select's dropdown will be replaced by the BottomSheet.
```

Reply:
Also a screenshot will help to visualize the usecase.

Reply:
Done.